extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-existing-openshift.ipynb

```diff
@@ -0,0 +1,352 @@
+                "    sys.exit('Please upgrade the Notebook dependency before you can proceed, you can do it by running the \"Reinstall Notebook dependencies\" command in command palette (View menu -> Command Palette…).')\n",
+                "def run_command(command):\n",
+                "    print(\"Executing: \" + command)\n",
+                "    !{command}\n",
+                "    if _exit_code != 0:\n",
+                "        sys.exit(f'Command execution failed with exit code: {str(_exit_code)}.\\n\\t{command}\\n')\n",
+                "    print(f'Successfully executed: {command}')"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "26fa8bc4-4b8e-4c31-ae11-50484821cea8",
+                "tags": [
+                    "hide_input"
+                ]
+            },
+            "outputs": [],
+            "execution_count": 1
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Set variables**\n",
+                "Generated by Azure Data Studio using the values collected in the Deploy Big Data Cluster wizard"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "e70640d0-6059-4cab-939e-e985a978c0da"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Check dependencies**"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "869d0397-a280-4dc4-be76-d652189b5131"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "run_command('kubectl version --client=true')\n",
+                "run_command('azdata --version')"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "c38afb67-1132-495e-9af1-35bf067acbeb",
+                "tags": []
+            },
+            "outputs": [],
+            "execution_count": null
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "### **Required information**"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "7b383b0d-5687-45b3-a16f-ba3b170c796e"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "invoked_by_wizard = \"AZDATA_NB_VAR_BDC_ADMIN_PASSWORD\" in os.environ\n",
+                "if invoked_by_wizard:\n",
+                "    mssql_password = os.environ[\"AZDATA_NB_VAR_BDC_ADMIN_PASSWORD\"]\n",
+                "    if mssql_auth_mode == \"ad\":\n",
+                "        mssql_domain_service_account_password = os.environ[\"AZDATA_NB_VAR_BDC_AD_DOMAIN_SVC_PASSWORD\"]\n",
+                "else:\n",
+                "    mssql_password = getpass.getpass(prompt = 'SQL Server 2019 Big Data Cluster controller password')\n",
```

Comment:
If this is actually intended to be ran outside the context of the wizard we should only be setting ACCEPT_EULA to true if it's run from the wizard and prompting the user otherwise.

Reply:
I think this has been discussed before with the PMs, and it is ok for us to do this.